unit-tests/func/fg/test-fg.py

```diff
@@ -0,0 +1,67 @@
+import pyrealsense2 as rs, test, ac
+
+dev = test.find_devices_by_product_line_or_exit(8)
```

Comment:
I think we should pass a string. `8` won't do.

Reply:
And either `devices` or `dev = ...[0]`

Reply:
"And either devices or device = ...[0]" ?